src/database/migrations/20240202032041_OrderApiRoles.ts

```diff
@@ -0,0 +1,25 @@
+import { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+  await knex.schema.alterTable('apiRoles', (table) => {
+    table.integer('order').defaultTo(100).notNullable();
+  });
+
+  const roleOrders = [
+    { role: 'MAPPER', order: 1 },
+    { role: 'GENERATOR', order: 2 },
+    { role: 'SHARER', order: 3 },
+    { role: 'ID_READER', order: 4 },
+  ];
+
+  for (const roleOrder of roleOrders) {
+    // eslint-disable-next-line no-await-in-loop
+    await knex('apiRoles').where('roleName', roleOrder.role).update({ order: roleOrder.order });
```

Comment:
The linter here is suggesting you store the promises in an array and await a Promise.all(..) of them.

Reply:
Sweet! That works. I did something similar and ran into issues with Knex and multiple requests but that seems to be working which is great!